dist/api_data.py

```diff
@@ -741,6 +741,16 @@ def __ge__(self, other):
             # !!! Must match WT_HS_FILE_MIN
             min='0')
         ]),
+    Config('heuristic_controls', '', r'''
+        control the behavior of various optimizations. This is primarily used as a mechanism for
+        rolling out changes to internal heuristics while providing a mechanism for quickly
+        reverting to prior behavior in the field''',
+        type='category', subconfig=[
+            Config('obsolete_tw_pages_dirty', '500', r'''
+                eviction to mark the number of obsolete time window pages that are marked as dirty
+                per btree in a single checkpoint''',
+                min=1, max=100000),
```

Comment:
If we want to be able to revert to the previous behaviour, the min should be 0.
```suggestion
                min=0, max=100000),
```